src/Uno.UI/UI/Xaml/DependencyObjectStore.cs

```diff
@@ -1629,11 +1629,11 @@ DependencyPropertyValuePrecedences newPrecedence
 				uiElt.OnPropertyChanged2(eventArgs);
 			}
 
+			OnDependencyPropertyChanged(propertyDetails, eventArgs);
```

Comment:
Here or a few lines earlier? c.f. https://github.com/unoplatform/uno/pull/6371

Reply:
Ah, that's embarrassing, I see I even approved the other PR and completely forgot about that 😀 . Anyway - this location is the correct one, I have updated my unit test with `Asserts` that check for current values in the individual phases and those asserts pass on UWP but would fail if `OnDependencyPropertyChanged` was called earlier.

Reply:
I wonder if we'll need to make this is a feature flag, as this may be breaking. We'll see what the unit tests are providing.

Reply:
I think it should not be a big problem, as it aligns with UWP behavior (and actually might fix some bugs, like was weird case with `NavigationView`). So far the test fails seem to be unrelated, so hopefully it will stay so 😀